### DISCUSSION TITLE
Task list, not applicable option

### DISCUSSION THREAD

**Discussion by Marvin-Brouwer** [community/community#110967] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I seem to be using task lists more often, mostly in PRs.
I primarily use these for DOD, however, sometimes they're just not applicable.

To communicate this I'd rather have a third option for something that's not applicable then just checking the box.

So something like: 

```[tasklist]
- [ ] Didn't do
- [x] Did do
- [~] Not applicable
```

Could look similar to: 

<img width="256" height="91" alt="image" src="https://github.com/user-attachments/assets/c84580a8-14ec-4372-a0de-4ce49bb42ef4" />

Or perhaps, keep it similar to GitLab: https://docs.gitlab.com/user/markdown/#task-lists.  

This way contributors communicate that the option is not applicable instead of skipped.
Showing clear intent.

**Comment by homersimpsons** [comment]
For the record, GitLab uses the `[~]` notation https://docs.gitlab.com/ee/user/markdown.html#task-lists.  Maybe it would be great to be consistent with the existing here.

**Reply by Marvin-Brouwer** [reply]
That sounds like a good idea to me

**Comment by Marvin-Brouwer** [comment]
I would still very much like to have this feature.

**Comment by Marvin-Brouwer** [comment]
Hi,

Just revisiting this suggestion to see if it’s still under consideration.  
It would be really helpful to have a “Not applicable” option for task lists, to clearly indicate items that don’t apply rather than leaving them unchecked.  

I also updated the original message to show what I mean more clearly, and I've referenced the GitLabs solution @homersimpsons commented. 

Any updates or feedback from the GitHub team would be greatly appreciated!

**Comment by Marvin-Brouwer** [comment]
Hi,

Just revisiting this suggestion to see if it’s still under consideration.
It would be really helpful to have this feature.

Any updates or feedback from the GitHub team would be greatly appreciated!
